docker-compose.yml

```diff
@@ -1,14 +1,9 @@
 version: "3"
-
 services:
   serge:
     image: serge
     stdin_open: true
     tty: true
     volumes:
-      - ./serge/data/:/var/serge/data
-      - ./translation_service/:/translation_service
-      - ./ingestion:/ingestion
-      - ./common:/common
-      - ./shared_directory:/shared_directory
-      - ./testing:/testing
```

Comment:
sorry about delay here. yes we are. We're just mounting all the data directly into its own `/var/tms` directory.
PR here: https://github.com/nyc-cto/tms/pull/17/files

Will be a cleaner implementation I think instead of having to decide what to copy over etc.